templates/server-acl-init-job.yaml

```
@@ -195,7 +195,7 @@ spec:
                 -create-client-token=false \
                 {{- end }}
 
-                {{- if .Values.global.acls.createReplicationToken }}
+                {{- if (or .Values.global.acls.createReplicationToken (and .Values.global.federation.enabled (and .Values.connectInject.enabled (and .Values.global.tls.enabled .Values.meshGateway.enabled)))) }}
```

I learnt recently that these `and` statements can be flattened.
```suggestion
                {{- if (or .Values.global.acls.createReplicationToken (and .Values.global.federation.enabled .Values.connectInject.enabled .Values.global.tls.enabled .Values.meshGateway.enabled)) }}
```

---

For my understanding, is it not enough to only check `.Values.global.federation.enabled` is enabled?